go-bindata-assetfs/main.go

```diff
@@ -54,16 +51,10 @@ func assetFS() *assetfs.AssetFS {
 	}
 	panic("unreachable")
 }`)
+	// Close files BEFORE remove calls (don't use defer).
```

Comment:
I see, there are two possible returns before closing the fds. The original code had a chance to leak also.

I see that doesn't matter in practice since the process ends, but I still think the code should be rewritten to avoid possible leaks just for better style, and to avoid encouraging others to write poor code.

> In UNIX there's no problem with closing FDs after deleting the backing file. I take it that windows complains when you do. Is that correct?

Yep, we need to find this out.